microsoft-edge/webview2/get-started/winui2.md

```diff
@@ -11,7 +11,7 @@ ms.date: 02/21/2023
 # Get started with WebView2 in WinUI 2 (UWP) apps
 
 In this tutorial, you:
-*  Set up your development tools for creating UWP apps that use WebView2 to display web content.
+*  Set up your development tools for creating UWP apps that use WebView2 to display web content. These instructions are also applicable to Xbox. 
```

Comment:
Should this be a separate note? Or a separate bullet point in this list?

Reply:
rendered article, intro at top:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/get-started/winui2?branch=pr-en-us-2897

Reply:
Result:
```
In this tutorial, you:
*  Set up your development tools for creating UWP apps that use WebView2 to display web content.
*  Create an initial WinUI 2 (UWP) app.
...

<!-- ------------------------------ -->
#### Platforms

This article applies to Windows and Xbox.

WinUI 2 supports UWP only.  These controls are backward-compatible.
```